examples/java/crd/package.json

```diff
@@ -7,12 +7,5 @@
     "build": "cdk8s import --language java",
     "synth": "mvn install && mvn compile && cdk8s synth",
     "test": "echo 'no test for java yet.'"
-  },
-  "dependencies": {
```

Comment:
Can't we leave these with the current latest version and let the dependency upgrade workflow take care of upgrading it? (i'm not sure why it didn't already...)

Reply:
It's much easier to manage this at the root level I believe. I'll also revisit the upgrade workflow shortly (via projen) after this first iteration is complete